consent-ui/src/app/auth/login/login.component.ts

```diff
@@ -47,6 +47,7 @@ export class LoginComponent implements OnInit {
 
   onSubmit() {
     localStorage.setItem(ApiHeaders.COOKIE_TTL, '0');
+    this.sessionService.clearStorage();
```

Comment:
I think we shouldn't do this, but instead, prefix every key with the session-id (authorization session-id).